### PR TITLE
Change default options

### PR DESCRIPTION
**Pull request by Honry**
- Set NHWC and WASM backend as default options
- Remove sw.js to disable sharedArrayBuffer support
- Fixed a bug in polyfill backend setup

### CODE REVIEW

**Comment by Honry** [comment]
@BruceDai, @huningxin, PTAL, thanks!

File: common/utils.js [hunk 1]
```diff
@@ -213,7 +213,8 @@ export async function setPolyfillBackend(device) {
   // Note: 'wasm' backend may run failed on some samples since
   // some ops aren't supported on 'wasm' backend at present
   const backend = device === 'cpu' ? 'wasm' : 'webgl';
-  const tf = navigator.ml.createContext().tf;
+  const ml = await navigator.ml.createContext();
```
Comment (BruceDai) [inline common/utils.js:216]: Here variable `ml` may not be descriptive. 
How about renaming it as `mlContext` or `context`?

**Review by BruceDai — COMMENTED** [review]
LGTM, and a nit of naming variable .

File: common/utils.js (hunk 1 above)
Comment (Honry) [reply common/utils.js]: Good catch! Thanks!
Fixed in new commit.

**Pull request merged**